src/tool_util.c

```diff
@@ -86,7 +86,7 @@ struct timeval tvnow(void)
     (void)gettimeofday(&now, NULL);
 #else
   else {
-    now.tv_sec = (long)time(NULL);
+    now.tv_sec = (time_t)time(NULL);
```

Comment:
time(NULL) should return a time_t and therefore this cast (and all others like it) shouldn't be needed anymore IMHO

Reply:
fix